core/iwasm/common/wasm_runtime_common.h

```diff
@@ -768,6 +772,10 @@ wasm_runtime_is_wasi_mode(WASMModuleInstanceCommon *module_inst);
 WASM_RUNTIME_API_EXTERN WASMFunctionInstanceCommon *
 wasm_runtime_lookup_wasi_start_function(WASMModuleInstanceCommon *module_inst);
 
+/* See wasm_export.h for description */
```

Comment:
Could you please add documentation for this function in `wasm_export.h`?

Reply:
done